gdal/doc/gdal_tutorial.dox

```diff
@@ -57,11 +57,10 @@ int main()
 
 In Python:
 \code
-    import gdal
-    from gdalconst import *
+    from osgeo import gdal, gdalconst
 
-    dataset = gdal.Open( filename, GA_ReadOnly )
-    if dataset is None:
+    dataset = gdal.Open(filename, gdalconst.GA_ReadOnly)
```

Comment:
I blindly took stuff from the original where gdalconst was imported and I assumed it was just for this constant. Your way sounds much more reasonable. Should I edit it or how does that work? :)